test/sharness/t0250-files-api.sh

```diff
@@ -812,15 +828,15 @@ tests_for_files_api "with-daemon"
 test_kill_ipfs_daemon
 
 test_expect_success "enable sharding in config" '
-  ipfs config --json Internal.UnixFSShardingSizeThreshold "\"1B\""
+  ipfs config --json Experimental.ShardingEnabled true
```

Comment:
@aschmahmann We need to decide on this.

Reply:
ideally I'd like to have a commit with just the new things and `Internal.UnixFSShardingSizeThreshold "\"1B\""` which is passing and then another commit where we drop it and all of the CIDs change and end up as the ones we already have as a way of verifying we've made the changes correctly (we can run the sharness tests locally to verify)

If this is too onerous we can probably leave this as is and I can do some manual verification before we merge